.flake8

```diff
@@ -61,6 +61,7 @@ per-file-ignores =
     ./ee/management/commands/migrate_clickhouse.py: T001
     ./ee/management/commands/run_async_migrations.py: T001
     ./ee/management/commands/backfill_persons_and_groups_on_events.py: T001
+    ./ee/management/commands/create_kafka_topics.py: T001
```

Comment:
We shouldn't be adding this here. There are always better solutions than print.

Reply:
All commands seem to use print as you see here hence the choice